lib/racer-client.coffee

```diff
@@ -17,10 +17,14 @@ class RacerClient
       cb null
       return
 
+    editor_file_path = editor.getPath().substring(0, )
+    editor_folder_path = editor_file_path.substring(0, editor_file_path.lastIndexOf("/"))
+
     tempOptions =
       prefix: "._racertmp"
     if @project_path
```

Comment:
Should we still test for the presence of a valid project path since we're not using it anymore?